src/contracts/Notification/UserInvitation.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\User\Notification;
+
+use Ibexa\Contracts\Core\SiteAccess\ConfigResolverInterface;
+use Ibexa\Contracts\User\Invitation\Invitation;
+use Symfony\Bridge\Twig\Mime\NotificationEmail;
+use Symfony\Component\Notifier\Message\EmailMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Notification\EmailNotificationInterface;
+use Symfony\Component\Notifier\Notification\Notification;
+use Symfony\Component\Notifier\Notification\SmsNotificationInterface;
+use Symfony\Component\Notifier\Recipient\EmailRecipientInterface;
+use Symfony\Component\Notifier\Recipient\SmsRecipientInterface;
+use Twig\Environment;
+
+final class UserInvitation extends Notification implements EmailNotificationInterface, SmsNotificationInterface
+{
+    private Invitation $invitation;
+
+    private ConfigResolverInterface $configResolver;
+
+    private Environment $twig;
+
+    public function __construct(
+        Invitation $invitation,
+        ConfigResolverInterface $configResolver,
+        Environment $twig
+    ) {
+        parent::__construct();
+
+        $this->configResolver = $configResolver;
+        $this->twig = $twig;
+        $this->invitation = $invitation;
+    }
```

Comment:
New PHP8 notation + empty `parent::__construct()` doesn't seem to be necessary:
```suggestion
    public function __construct(
        private readonly Invitation $invitation,
        private readonly ConfigResolverInterface $configResolver,
        private readonly Environment $twig
    ) {
    }
```

Reply:
It is empty now, but if parent add something in the constructor itself then our notification would be not properly built. I would say, that calling parent constructors is mandatory, even when at this point is doing nothing unless you are overriding specific behaviour.